analyzer/src/funTest/assets/projects/synthetic/spdx/project/project.spdx.yml

```diff
@@ -14,7 +14,8 @@ documentDescribes:
 packages:
 - SPDXID: "SPDXRef-Package-xyz"
   description: "Awesome product created by Example Inc."
-  copyrightText: "Copyright (C) 2020 Example Inc."
+  copyrightText: "Copyright (C) 2020 Example Inc.
```

Comment:
copyrightText is a free text field like description in line 27 so I propose you indent it the same and use a `\` to break lines.

```
copyrightText: "Copyright (C) 2020 Example Inc. \
    Copyright (C) 2021 Second Example Inc."
```

Reply:
Will do. 
Just a question while we are at it: I haven't found a mention / convention in the guidelines of SPDX on how to deal with several copyright statements in one `copyrightText`. 
There is a mention that it can stretch over multiple lines, but no mention on how to handle several copyrights. Did I miss anything in the guidelines? Otherwise that would be a discussion I would like to start in the SPDX tech list.